README.md

```diff
@@ -1,2 +1,142 @@
 [![Work in Repl.it](https://classroom.github.com/assets/work-in-replit-14baed9a392b3a25080506f3b7b6d57f295ec2978f6f33ec97e36a161684cbe9.svg)](https://classroom.github.com/online_ide?assignment_repo_id=4682119&assignment_repo_type=AssignmentRepo)
 # Careers in Computer Science
+## Software Developer
+### [Responsible for the design, development and implementation of new or modified software products] (www.roberthalf.com.au/employers/it-technology/software-developer-jobs)
```

Comment:
Formatting of this part seems off.  Why is it a level 3 header?  As a clickable link, there must be no space between the [] and ().
```
[Responsible for the design, development and implementation of new or modified software products](www.roberthalf.com.au/employers/it-technology/software-developer-jobs)```